Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -129,4 +130,12 @@ RefPtr<Gfx::Bitmap> Layer::try_copy_bitmap(Selection const& selection) const
     return result;
 }
 
+void Layer::erase_selection(Selection const& selection)
+{
+    Gfx::Painter painter { bitmap() };
+    Gfx::IntRect image_and_selection_intersection = m_image.rect().intersected(selection.bounding_rect());
+    Gfx::IntRect translated_to_layer_space = image_and_selection_intersection.translated(-location());
```

Comment:
You can use `auto` (or even better, `auto const` here. We prefer using `auto` here instead of explicitly naming types, especially in cases like this where the types are easily inferred.